### PR TITLE
Compare structure fix merge conflicts

### PR DESCRIPTION
**Pull request by Smrosner**

### CODE REVIEW

**Comment by Smrosner** [comment]
just added some overall file structure, still have to write tests.

**Review by melliottgithub — COMMENTED** [review]
Try to add some tests and more explanation to all the changes you are making

**Review by melliottgithub — APPROVED** [review]

File: client/App.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Compare from './components/Compare';
+import Compare from './modules/Compare';
```
Comment (Smrosner) [inline client/App.js:2]: switch path to use module component folder

File: client/modules/Compare/components/Carousel/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (Smrosner) [inline client/modules/Compare/components/Carousel/Carousel.js:1]: Build out Carousel import/export and inner component structure

File: client/modules/Compare/components/Carousel/Carousel.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      <div>
+        <Section />
+        <LeftButton />
+        <Card />
+        <RightButton />
+      </div>
+    )
+}
+
+export default Carousel;
```
Comment (Smrosner) [inline client/modules/Compare/components/Carousel/Carousel.js:18]: Build out Carousel import/export and inner component structure

**Comment by Smrosner** [comment]
additional explanation: build out Carousel, Card, CardAssets,  CardDetails, import/export, and corresponding inner FavIcon, Image, Category, ProductName, Price, and AvgRating component structures.

**Pull request merged**